step-controller/step-controller-server/src/main/java/step/controller/services/entities/AbstractEntityServices.java

```diff
@@ -197,4 +198,35 @@ protected <R> AsyncTaskStatus<R> scheduleAsyncTaskWithinSessionContext(AsyncTask
     public TableResponse<T> request(TableRequest request) throws TableServiceException {
         return tableService.request(entityName, request, getSession());
     }
+
+    @Operation(operationId = "get{Entity}History", description = "Retrieves entity's versioned")
```

Comment:
entity's versions.... or
Retrieves the versions of the entity with the given id...

Reply:
changes made